source/features/copy-file.tsx

```diff
@@ -22,18 +23,28 @@ function handleClick({delegateTarget: button}: delegate.Event<MouseEvent, HTMLBu
 }
 
 function renderButton(): void {
-	for (const button of select.all('.file-actions .btn, [data-hotkey="b"]')) {
-		button
-			.parentElement! // `BtnGroup`
-			.prepend(
-				<button
-					className="btn btn-sm tooltipped tooltipped-n BtnGroup-item rgh-copy-file"
-					aria-label="Copy file to clipboard"
-					type="button"
-				>
-					Copy
-				</button>
-			);
+	for (const button of select.all<HTMLAnchorElement>([
+		'.file-actions .btn[href*="/raw/"]', // `isGist`
+		'[data-hotkey="b"]'
+	])) {
+		const copyButton = (
+			<button
+				className="btn btn-sm tooltipped tooltipped-n BtnGroup-item rgh-copy-file"
+				aria-label="Copy file to clipboard"
+				type="button"
+			>
+				Copy
+			</button>
+		);
+		if (pageDetect.isGist() && button.href.endsWith('.svg')) { // SVG files have a render blob button
+			button.before(copyButton);
+		} else {
+			button
+				.parentElement! // `BtnGroup`
+				.prepend(copyButton);
+		}
+
+		groupSiblings(button);
```

Comment:
Without this (effects only gists)

![image](https://user-images.githubusercontent.com/16872793/86920524-eafb1500-c0f7-11ea-918d-32d99edac218.png)